chain/default_syncer_test.go

```diff
@@ -613,7 +629,7 @@ func TestLoadFork(t *testing.T) {
 
 	fakeChildParams.Nonce = uint64(2)
 	forklink1blk3 := th.RequireMkFakeChild(t, fakeChildParams)
-	//th.FakeChildParams{Parent: forkbase, GenesisCid: genCid, StateRoot: genStateRoot, Nonce: uint64(2)})
```

Comment:
Id say remove this commented out FakeChildParams statement rather than updating

Reply:
👍